keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -1105,6 +1105,24 @@ The additional benefits of this approach comparing to initial proposal:
   same time as before and can utilize as much of the graceful termination period as they need. The Pod graceful
   termination period semantic also stay unchanged.
 
+### Restarting sidecar containers during Pod termination
+
+Sidecar containers should be restarted when a Pod is terminated to ensure that they
+are running while the main containers are running. This feature was originally planned
```

Comment:
I think its both main containers and other sidecars that should terminate first.  Not sure I like this language any better, but:

```suggestion
are running while the containers that should terminate prior to its termination are still running. This feature was originally planned
```